libs/remix-ui/run-tab/src/lib/actions/index.ts

```diff
@@ -26,6 +28,13 @@ export const initRunTab = (udapp: RunTab) => async (reducerDispatch: React.Dispa
   setupEvents(plugin, dispatch)
   setInterval(() => {
     fillAccountsList(plugin, dispatch)
+    if (plugin.REACT_API?.instances?.instanceList?.length){
```

Comment:
couldy you wrap this in ta function like `fillAccountsList`?

Reply:
done